tests/test_util.ts

```diff
@@ -1,6 +1,105 @@
-import { Redis, connect, RedisConnectOptions } from "../redis.ts";
-import { assert } from "../vendor/https/deno.land/std/testing/asserts.ts";
+import { connect, Redis, RedisConnectOptions } from "../redis.ts";
 import { delay } from "../vendor/https/deno.land/std/async/mod.ts";
+import { assert } from "../vendor/https/deno.land/std/testing/asserts.ts";
+
+type TestFunc = () => void | Promise<void>;
+type TestServer = {
+  path: string;
+  process: Deno.Process;
+};
+
+export class TestSuite {
+  private encoder = new TextEncoder();
+
+  private tests: { name: string; func: TestFunc }[] = [];
+  private beforeEachs: TestFunc[] = [];
+  private afterAlls: TestFunc[] = [];
+
+  constructor(private prefix: string) {}
+
+  startRedis = async ({
```

Comment:
I think `TestSuite` should not have methods related to Redis.
Is it possible to change this method to a normal function?